backend/dataall/core/permissions/db/__init__.py

```diff
@@ -1,14 +0,0 @@
-import logging
```

Comment:
How is this operation handled now? Is there any impact for newly deployed data.all infra?

Reply:
I moved the definition to the new class `TenantPolicyService`, as it seems to belong there.

This function is called in `backend/api_handler.py` and `backend/local_graphql_server.py` as it was before.